### PR TITLE
Removes BoxStation in favor of IceBox

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Its that time again. Since icebox is now merged and seems to be working well, I propose we remove box once and for all. 

Makes metastation the new default fallback map.

## Why It's Good For The Game
Right now map changes to box or ice box are having to be mirrored to the other map, leading to lots of extra work involved in making improvements.

## Changelog
:cl: Skoglol
del: BoxStation has been removed in favor of IceBox. 
/:cl:

### CODE REVIEW

**Comment by bobbahbrown** [comment]
metastation as default, as it was always destined to be

**Comment by Tlaltecuhtli** [comment]
can we remove ice box station in favour of ice pubby

**Comment by pireamaineach** [comment]
Box is the map /tg/ launched with, come on gamer. Icebox isn't even that much more of an improvement people just like it because snoe pwetty

**Comment by kriskog** [comment]
Box is old, its shit, its got nothing going for it. Ice box at least adds multi-z and a gimmick, and is added by a mapper that isnt gone yet. You wanted box improved to be kept around? Thats Ice box. You cant have it both ways.

**Comment by Whoneedspacee** [comment]
> Box is old, its shit, its got nothing going for it. Ice box at least adds multi-z and a gimmick, and is added by a mapper that isnt gone yet. You wanted box improved to be kept around? Thats Ice box. You cant have it both ways.

>mapper

 🤣 i wish i was that good

**Comment by mrhugo13** [comment]
Before the redditors get here, this is fucking epic.

Box has been old and shit for such a long time with no redeeming quality other then "it was the map /TG/ launched with" while being a huge pain in everyones ass, with icebox coming around, there's pretty much no reason to keep the old box around because there's way better alternatives.

Icebox has a better theme, has someone to actually willing to maintain it, and has an interesting spin that actually works. Old box now is just this obsolete thing that barely any mapper wants to actually work with because of this fact, hell i still fucking hated box despite the fact i spent 2 PR's just fucking with it's powergrid and pipenet to make it more managable on a single layer.

I'm hoping this PR won't be subject to the last one where most of it ended up being "Oh i promise i will take good care of box if you just let me have it for a little longer!" with everyone quickly forgetting about that once the PR closed.

TL;DR: Box sucks and icebox is actually worth spending time on.

**Comment by EOBGames** [comment]
Thank fuck.

Mirroring changes between the two stations sucks.

**Comment by Cyberboss** [comment]
I will defend Boxstation's honor till the day I die

**Comment by JenqaDev** [comment]
shut the fuck up we want jungle meta and desert delta

**Comment by anconfuzedrock** [comment]
I think we should give icebox the ai core in the center in this case. it makes more sense anyways because having a satellite when that satellite is reached by just wearing a coat and walking past escape is pretty stupid.

**Comment by Whoneedspacee** [comment]
just give icebox ai some machine gun turrets on the outside bam we cool

**Comment by Quietly12** [comment]
After 15 years of faithful service, Centcom found that the endlessly mounting power and design issues with one of their first space stations required a drastic overhaul, or a full decommissioning. 

Instead of abandoning the station that so many loved; instead it was brought into the nearest suitable gravity well. An endeavor that brought it safely to rest on an uncharted icy moon, where it could be freely expanded and repaired, anticipating a future where it may one day return to the stars more robust than ever.

**Comment by kriskog** [comment]
This isnt a PR proposing changes to either of the maps. If you have improvements or fixes for ice box, feel free to PR it or talk someone else into it. Not here though, its out of scope and is not a prerequisite for this removal.

**Comment by JenqaDev** [comment]
> This isnt a PR proposing changes to either of the maps. If you have improvements or fixes for ice box, feel free to PR it or talk someone else into it. Not here though, its out of scope and is not a prerequisite for this removal.

I'm 200% sure it was a normal joke and he wasn't actually telling you to add those things. Ha Ha
I can't wait for the day that someone adds machine guns to regular rounds and lets us craft them using 10  disablers and wired up to the APC for that tasty power suc

**Comment by JenqaDev** [comment]
About the one before that he was just sharing his thoughts, he didn't ask you to add anything at all.

**Comment by pireamaineach** [comment]
All of the issues with box are superficial. The layout of box is fine, I don't see why everyone shits on Box but refuses to fix it's issues.

**Comment by LemonInTheDark** [comment]
That's why we have icebox tho right? one is maintained, the other isn't.

I like box, but the mappers I know keep complaining about updating 2 maps with 1 change, and I'm inclined to agree. We should have one map that's "box"

**Comment by pireamaineach** [comment]
Rename Icebox to Box. If there is only one then the variant name isn't needed.

**Comment by pireamaineach** [comment]
Still, I'd prefer old box persisting.

**Comment by Fox-McCloud** [comment]
Not a big fan of this.

Box is old, but it's been through many upgrades and iterations; I'd venture to guess also (though stats will need to be broken out) that it's the most popular map, overall. 

This isn't to say we have to keep old things because they're old---but rather that the reason something has made it *this far* is likely because it has something good going for it. 

Icebox personally rubs me the wrong way a bit---it's an interesting design, to be sure, but it's distinctively *not* Space Station 13; it's more akin to its own thing than SS13, what with the fact that it's...well, snow---snow everywhere---on a planet.

This isn't to say that Ice Box shouldn't exist or it's bad---just that it's a bit of a novelty/outlier compared to the other stations, and I don't believe it should personally replace the primary, flagship station TG has always had.

**Comment by Whoneedspacee** [comment]
icebox literally changes like a couple things with the game is still almost entirely the same game so i dont really get that aspect of your argument

box is popular on other servers for sure but on tg for sure it is not

**Comment by zxaber** [comment]
Box is part of the game's history, and an important memory for a lot of people, I get that.

But it's time to let it go.

Nothing about current Box is unique or interesting. With Metastation being the more popular map among a majority of players, Box gets left behind. It plays like a worse, rotated Meta, with nothing to give it any reason to exist. Icebox at least has a fun gimmick that defines the map.

**Comment by kassori** [comment]
Support. Icebox is my excuse to get off and do something productive.

**Comment by pireamaineach** [comment]
> Box is part of the game's history, and an important memory for a lot of people, I get that.
> 
> But it's time to let it go.
> 
> Nothing about current Box is unique or interesting. With Metastation being the more popular map among a majority of players, Box gets left behind. It plays like a worse, rotated Meta, with nothing to give it any reason to exist. Icebox at least has a fun gimmick that defines the map.

Saying Meta is unique and interesting in contrast to Box is a pretty bizaare take. People are bored of Meta, it's ridiculously overplayed and elicits groans half the time it's picked. Icebox's gimmick is pretty shallow and bad, and Icebox doesn't fix any of the core issues of Box.

**Comment by bobbahbrown** [comment]
Both @Fox-McCloud and @pireamaineach are very clearly shown to be disconnected from the playerbase when they assume meta isn't the most popular map. We had to implement a forced map rotation because people would just keep playing meta.

Don't just say uhh people don't like the map or uhh people surely like box more without having some actual substance to your statements, though I appreciate fox at least said that he would need to see some numbers to have any substance to his claim.

**Comment by zxaber** [comment]
>Saying Meta is unique and interesting in contrast to Box is a pretty bizaare take.

That was not the point whatsoever. Box and Meta are more or less the same map. Given that, we don't need both, and Meta is so popular that (as said by the bee above) we had to force people to play other maps. Box honestly should have been dropped years ago.

Icebox adds uniqueness to the Box layout.

**Comment by pireamaineach** [comment]
I didn't say people like Box more I simply sat that Meta is boring. The idea that Meta is somehow more unique than Box bewilders me. I didn't say "OH EVERYONE LIKE BOX MORE"

**Comment by EOBGames** [comment]
With regards to fixing box/icebox, keeping both maps is only gonna make that problem far worse, as nobody's particularly enthralled by the idea of having to mirror changes to both maps. Already I've found issues with sync between the two maps, and frankly if we don't make a choice between them soon I'd see updates to both stopping entirely.

**Comment by pireamaineach** [comment]
I'd rather have Box, a legacy map, then some weird testbed map that frankly from what I hear people don't find that fun. It's a weird  move when they have the same core issues.

**Comment by Whoneedspacee** [comment]
icebox literally changes nothing with box practically its just fucking box on ice moon my dude i dont know how it could possibly be worse than box

im not gonna make an entirely different station map on top of remaking lavaland lol

**Comment by EOBGames** [comment]
> icebox literally changes nothing with box practically its just fucking box on ice moon my dude i dont know how it could possibly be worse than box
> 
> im not gonna make an entirely different station map on top of remaking lavaland lol

Yeah you've got someone else to do it for you

**Comment by LemonInTheDark** [comment]
he's showing his hand, quick take screenshots

**Comment by Capsandi** [comment]
If this pr gets merged now, your Chilledstation will have a tougher reception due to the added weight of removing, or preparing to remove a station layout which a portion of the playerbase prefers over most current maps. You could circumvent this by moving box back to space in an accompanying pr, the only arising issue i see with this is the possibility of box having enough large changes to further accommodate it to the moon that it would considerably delay the release of the chilled map, though i don't believe this is likely to be an issue

**Comment by optimumtact** [comment]
image

**Comment by Learner10** [comment]
Box is practically no different from Icebox. You've got 2-3 modified areas on station with a few added changes here and there, and if you step outside, instead of popping like a plump cherry in 0 pressure (and suffocating), you just freeze to death (and suffocate). Maybe you even fall down a hole, break your legs, AND freeze to death. Is the argument against Icebox that you can't walk into space and get lost forever?

As for the argument for "it was first"... current BoxStation is rather quite different from how it was from the start. I'd counterargue with the philosophy that "a broom who's brush and handle have been replaced is not the same broom". At best, it's got some departments in some of the same places. Today's Box isn't really the same thing that OldBox was.

**Comment by pireamaineach** [comment]
Yeah wait, that's actually a really solid argument. Are we going to have 2 maps with this gimmicky ice planet gimmick ?

If Icebox is destined to be replaced by ChilledStation, why bother removing Box for it's sake ?

**Comment by LemonInTheDark** [comment]
Chilled is a good ways out

**Comment by EOBGames** [comment]
> Yeah wait, that's actually a really solid argument. Are we going to have 2 maps with this ice planet gimmick ?
> 
> If Icebox is destined to be replaced by ChilledStation, why bother removing Box for it's sake ?

Because it's at least half a year out if not longer, and that time would require both box and icebox to be maintained simultaneously, which, as previously stated, is horrible.

**Comment by pireamaineach** [comment]
Honestly I'd rather remove Icebox, or just stop cross-maintenance. Both are preferable, to my mind, as someone who, isn't working on it, so take it with a grain of salt, to later having to port Icebox back to spess.

**Comment by Whoneedspacee** [comment]
but im working on icebox

**Comment by kriskog** [comment]
You also don't play here, so I already took your opinion with a good pinch of salt.

Just because someone has fantasized about replacing icebox doesnt mean it will actually happen. For one, work on that mythical replacement isnt nearly done and even when it is nothings to say it will get accepted as a replacement. 

> Box is old, but it's been through many upgrades and iterations; I'd venture to guess also (though stats will need to be broken out) that it's the most popular map, overall.

Player vote held a few months ago landed box on 3rd for keeping around, only beating pubby which is notoriously hated. This was pre ice box.
Right now, box is the 3rd most played map, losing by good margin to meta and delta who make up around 60% of maps played.

**Comment by pireamaineach** [comment]
First of all, I do play here, it's the only server I play.

Secondly, I've never once heard someone complain about Pubby, much less would I consider it "notoriously hated."

**Comment by bobbahbrown** [comment]
> First of all, I do play here, it's the only server I play.
> 
> Secondly, I've never once heard someone complain about Pubby, much less would I consider it "notoriously hated."

the last time you played was 3 months ago and really more realistically 4 months ago, why are you coming on this pr to complain when you don't play

**Comment by pireamaineach** [comment]
I played today. I don't play all that often but being dismissive because of your preconceived notion that I don't play is ridiculous.

**Comment by Learner10** [comment]
The main issue with Boxstation vs Metastation are their similarities, both in shape, and department locale. This was one of the previous and strongest arguments against BoxStation, and now there's the fact that we have an even more similar map known as Icebox, and that maintaining both maps with changes is becoming an issue. It took putting BoxStation on an ice moon to make it stand apart from Metastation.

Also, I don't quite think Pubby is as hated as you think. It's rather liked for being aesthetic. Like... it doesn't particularly stand out, but it does stand apart, which is fine. It's a different map after all.

**Comment by kriskog** [comment]
I like pubby, sadly many others do not. Its largely due to its pretty unique layout, and not playing and exploring it enough to learn.

**Comment by optimumtact** [comment]
image

**Comment by Fabian272** [comment]
Jeez if you hate it that much just take it out of rotation till its fucking fixed but still. "Shits broken time to delete it"

**Comment by mrhugo13** [comment]
> 
> 
> Jeez if you hate it that much just take it out of rotation till its fucking fixed but still. "Shits broken time to delete it"

Problem is that would pretty much give people no reason to maintain a map that is both garbage and out of rotation.

**Comment by FloranOtten** [comment]
The only argument for keeping Box is "It was first!" and that could apply to box as well as the fucking telegraph. Times change, box sucks, if you want confirmation of that just look at what map people actually play.

**Comment by trollbreeder** [comment]
> > Jeez if you hate it that much just take it out of rotation till its fucking fixed but still. "Shits broken time to delete it"
> 
> Problem is that would pretty much give people no reason to maintain a map that is both garbage and out of rotation.

Sadly this was the approach for Kilostation. It was taken out, everyone forgot it existed, nobody fixed the crash issues, then it was shot in the head and dumped in the river by the map vote.

**Comment by drIcepop** [comment]
Box is better than icebox

**Comment by kassori** [comment]
"benefits" icebox has over box
Its got a single ghost role
Its easy cheese for space dragon, blob(?), xenos
Its got stupid high mob density, I counted like 400 
throwing people out of airlocks doesn't do much
Less ruins, more danger, less reason to explore

Please don't say it literally changes nothing

**Comment by zxaber** [comment]
The real question is what does Box bring to the game? It's a worse version of Metastation. Is there anything unique about it that warrent the upkeep cost, aside from pure nostalgia?

Icebox may be the catalyst of this change, but I argue this is a PR we should have done a long time ago.

**Comment by Quietly12** [comment]
Box is currently broken, and although many comments seem to believe people like it, they've probably not played in a box round recently enough to see the constant complaining/anger in chat from the various departments that are screwed over for an entire shift because players voted box.

As it stands the only results of this PR will be
1). Box is not removed and someone has to overhaul the entire map to make it playable before anyone can actually play it
2). Box is removed because there's no point in overhauling it when icebox works.

**Comment by Kaffe-work** [comment]
when I get box as CE I STILL want to kill myself because the powernet sometimes just gives up, some wires burn up and it's incredibly hard to search and locate every fucking thing that is broken.

**Comment by Tlaltecuhtli** [comment]
"we need box because reasons" t. 6-7 people who dont play or dont even contribute

**Comment by 81Denton** [comment]
conflict

**Review by 81Denton — APPROVED** [review]
I agree with the arguments that have been made in favor of replacing box with icebox - especially maintainability and redundance. Box has been around for ages, but that shouldn't be a factor in deciding whether we keep it or not.

**Comment by pireamaineach** [comment]
Isn't that a little bit short sighted considering future plans for Icebox.

**Comment by Whoneedspacee** [comment]
who said we 100% cant have two ice maps

we'll have to see what icebox is like when chilled comes along and see what chilled brings to the table maybe brings out the multiz aspects better
